_content/articles/summaries-of- the- dharma-dn-12_analayo.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Summaries of the Dharma: A Translation of Dīrgha-āgama Discourse No. 12"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/summaries.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1faTaaSYl4nZfzCjpUmp6Vo30h_PrqhwI/view?usp=drive_link"
+course: karma
+tags:
+  - agama
+  - ethics
+year: 2013
+journal: "Asian Literature and Translation"
+publisher: "Cardiff University Press"
+volume: 1
+number: 6
+pages: "1--14"
+---
+
+A translation of the twelfth discourse in the Chinese Dīrgha-āgama with a short introduction by Bhikkhu Anālayo. This discourse is without known parallels, though it does share similarities with the Saṅgīti-sūtra and the Daśottara-sūtra.
```

Comment:
What did you learn from the sutta?  For me, it's a striking example of how the path karma truly is "neither light nor dark"

Reply:
I put something in that stuck out to me. I can remove it if it doesn't fit or seem correct.